third_party/CMakeLists.txt

```diff
@@ -123,21 +131,32 @@ endif()
 
 add_subdirectory(ObjexxFCL)
 set_target_properties(objexx PROPERTIES FOLDER ThirdParty/ObjexxFCL)
+
 add_subdirectory(BCVTB)
 set_target_properties(bcvtb PROPERTIES FOLDER ThirdParty/BCVTB)
+
 add_subdirectory(Expat)
 set_target_properties(epexpat PROPERTIES FOLDER ThirdParty/Expat)
+
 add_subdirectory(FMI)
 set_target_properties(epfmiimport PROPERTIES FOLDER ThirdParty/FMI)
+target_link_libraries(epfmiimport turn_off_warnings)
```

Comment:
Turn off warnings for some of the third party libs that were issuing some:
* FMI
* zlib
* DEligh